lib/sasl/src/sasl.appup.src

```diff
@@ -35,12 +35,16 @@
   {<<"^4\\.1$">>,[restart_new_emulator]},
   {<<"^4\\.1\\.0(?:\\.[0-9]+)+$">>,[restart_new_emulator]},
   {<<"^4\\.1\\.1(?:\\.[0-9]+)*$">>,[restart_new_emulator]},
-  {<<"^4\\.1\\.2(?:\\.[0-9]+)*$">>,[restart_new_emulator]}],
+  {<<"^4\\.1\\.2(?:\\.[0-9]+)*$">>,[restart_new_emulator]},
+  {<<"^4\\.2$">>,[restart_new_emulator]}
+],
  [{<<"^4\\.0$">>,[restart_new_emulator]},
   {<<"^4\\.0\\.0(?:\\.[0-9]+)+$">>,[restart_new_emulator]},
   {<<"^4\\.0\\.1(?:\\.[0-9]+)*$">>,[restart_new_emulator]},
   {<<"^4\\.0\\.2(?:\\.[0-9]+)*$">>,[restart_new_emulator]},
   {<<"^4\\.1$">>,[restart_new_emulator]},
   {<<"^4\\.1\\.0(?:\\.[0-9]+)+$">>,[restart_new_emulator]},
   {<<"^4\\.1\\.1(?:\\.[0-9]+)*$">>,[restart_new_emulator]},
-  {<<"^4\\.1\\.2(?:\\.[0-9]+)*$">>,[restart_new_emulator]}]}.
+  {<<"^4\\.1\\.2(?:\\.[0-9]+)*$">>,[restart_new_emulator]},
+  {<<"^4\\.2$">>,[restart_new_emulator]}
+]}.
```

Comment:
The `sasl.appup.src` file is automatically updated based on automatically selected versions, so you should not update it.